.npmignore

```diff
@@ -6,3 +6,12 @@ Dockerfile
 Makefile
 docker-compose.yml
 webpack.config.js
+npm-debug.log
+jest
+.DS_Store
+.*.swp
```

Comment:
Do we need to ignore vim files?

Reply:
@jelohe I've added the entries that the .npmignore has (as we are overriding it) + our files.